TCAT.xcodeproj/project.pbxproj

```diff
@@ -1093,7 +1093,7 @@
 				INFOPLIST_FILE = TCAT/Supporting/Info.plist;
 				INFOPLIST_KEY_CFBundleDisplayName = "Ithaca Transit";
 				INFOPLIST_KEY_LSApplicationCategoryType = "public.app-category.travel";
-				IPHONEOS_DEPLOYMENT_TARGET = 15.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 18.0;
```

Comment:
Same here, maybe keep it as 15 unless there is a specific reason we needed up upgrade